SwiftUI_Homework/SwiftUI_Homework/TabView/BasketView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  BasketView.swift
+//  SwiftUI_Homework
+//
+//  Created by Анастасия Козлова on 22.01.2023.
+//
+
+import SwiftUI
+
+/// Экран "Корзина"
+struct BasketView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let buyText = "Buy"
+        static let buyYesText = "Вы уверенны, что хотите купить этот тариф?"
+        static let yesText = "Да"
+        static let noText = "Нет"
+        static let blueColor = Color(#colorLiteral(red: 0.2569369539, green: 0.8115655123, blue: 0.9686274529, alpha: 0.4798957875))
+        static let widthButtonValue: CGFloat = 240
+        static let heightButtonValue: CGFloat = 50
+        static let cornerRadiusValue: CGFloat = 15
+        static let paddingButtonValue: CGFloat = 20
+    }
+    
+    // MARK: - Body
+    
+    var body: some View {
+        ZStack {
+            Color.black.ignoresSafeArea(edges: .top)
+            VStack {
+                Text(userBuy.rate)
+                    .foregroundColor(Color.white)
+                    .padding(Constants.paddingButtonValue)
+                    .font(Font.largeTitle)
+                buyButtonView
+            }
+        }
+    }
+    
+    // MARK: - Environment
+    
+    @EnvironmentObject private var userBuy: UserBuy
+
+    // MARK: - State
```

Comment:
```suggestion
    // MARK: - State
```
Женя сказал что можно отдельно это все не выносить, а сделать просто приват проперти

Reply:
у тебя на некоторых экранах так сделано